README.md

```diff
@@ -145,7 +145,7 @@ Otherwise the Ubuntu package `python-is-python3` provides the `python` alias.
 1. Install dependencies via [Homebrew](https://brew.sh/):
 
 ```shell
-brew install python@3.10 gcc cmake ninja libomp netcdf-cxx eigen nlohmann-json protobuf lapack git open-mpi
+brew install python@3.10 gcc cmake ninja libomp netcdf-cxx eigen nlohmann-json protobuf git open-mpi
```

Comment:
(non-expert for compiling on Mac writing here) Could it ever occur that the Accelerate framework (which I assume would provide LAPACK?) is not installed on a MacOS system? If that can happen, maybe we need to add a word of warning that the homebrew `lapack` package would be needed in that case?

Reply:
No it's a system library. That's also the reason why the previous cmake builds weren't finding lapacke from homebrew. FindLAPACK was most likely finding the Accelerate framework. To get the version in home-brew, you would have needed to set `CMAKE_PREFIX_PATH` to the install location for home-brew and specify the `BLA_VENDOR `.

Reply:
OK, then lets leave it as-is. @eguiraud-pf OK for you as well?